changelog/changes/center-carpet-useful.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Improved node robustness"
+type: feature
+authors: tobim
+pr: 5233
+---
+
+Pipelines that are running in a node are now partially moved to a subprocess for
+improved error resilience and resource utilization. Operators that need to
+communicate with a component still run inside the main node process for
+architectural reasons. You can set `tenzir.disable-forked-pipelines: true` in
```

Comment:
Where is this option handled?

Reply:
in `spawn_node.cpp` and `node.cpp`.
I didn't see any good reason to make it part of the `pipeline_executor` interface. I'm not sure we should change this before the shift to the new execution model.